ui_framework/components/tabs/_tabs.scss

```diff
@@ -23,6 +27,11 @@
   border-radius: 0; /* 1 */
   margin-bottom: -1px; /* 3 */
 
+  @include darkTheme {
+    background-color: $tabBackgroundColor--darkTheme;
+    border-color: $tabBorderColor--darkTheme;
```

Comment:
Oh OK cool, I see. I took a look in the browser and it looks like the text color has poor color contrast against the background:

![image](https://user-images.githubusercontent.com/1238659/28832738-2c7f2650-7692-11e7-90af-18559e9bae04.png)

I think we should address this color contrast issue. How about using a different variable -- can we add `color: $tabColor--darkTheme;`? This will get it to look like this:

![image](https://user-images.githubusercontent.com/1238659/28832792-4a955506-7692-11e7-94d6-d04f79f39ca7.png)